tests/tests_lite/plugins/precision/test_deepspeed.py

```diff
@@ -50,16 +50,12 @@ def test_deepspeed_precision_backward():
 
 
 @RunIf(deepspeed=True)
-def test_deepspeed_engine_is_steppable():
+@mock.patch("deepspeed.DeepSpeedEngine", autospec=True)
+def test_deepspeed_engine_is_steppable(engine):
```

Comment:
Agree. Since the test is just about checking comparing the signature, the autospec is enough.